js/jquery.utils.js

```diff
@@ -416,11 +416,18 @@ function scrollToElement($element, duration) {
     var viewportBottom = viewportTop + $window.height();
     var elementTop = $element.offset().top;
     var elementBottom = elementTop + $element.height();
-    
-    if ( elementBottom > viewportBottom || elementTop < viewportTop ) {                                
-        // Element is not fully visible - scroll page to the element
-        $('body').animate({
+
```

Comment:
Can you explain what the purpose of the following line is `if ( elementBottom > viewportBottom || elementTop < viewportTop ) {` and why you think it should be removed?